tests/swarm-deploy/test_frontend_served.py

```diff
@@ -0,0 +1,51 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+from typing import List
+
+import pytest
+import requests
+import tenacity
+from docker.models.services import Service
+from pytest_simcore.helpers.constants import MINUTE
+from tenacity.stop import stop_after_delay
+from tenacity.wait import wait_fixed
+from yarl import URL
+
+
+@pytest.mark.parametrize(
+    "test_url,expected_in_content",
+    [
+        ("http://127.0.0.1:9081/", "osparc/boot.js"),
+        ("http://127.0.0.1:9081/s4l/index.html", "Sim4Life"),
+        ("http://127.0.0.1:9081/tis/index.html", "TI Treatment Planning"),
+    ],
+)
+def test_product_frontend_app_served(
+    simcore_stack_deployed_services: List[Service],
+    traefik_service: URL,
+    test_url: str,
+    expected_in_content: str,
+    loop,
+):
+    # NOTE: it takes a bit of time until traefik sets up the correct proxy and
+    # the webserver takes time to start
+    # TODO: determine wait times with pre-calibration step
+    @tenacity.retry(
+        wait=wait_fixed(5),
+        stop=stop_after_delay(1 * MINUTE),
+    )
+    def request_test_url():
+        resp = requests.get(test_url)
+        assert (
+            resp.ok
+        ), f"Failed request {resp.url} with {resp.status_code}: {resp.reason}"
+        return resp
+
+    resp = request_test_url()
+
+    # TODO: serch osparc-simcore commit id e.g. 'osparc-simcore v817d82e'

```

Comment:
Are you sure this would be in the header? Form what I remember in the index.html you are not able to find that information.

Reply:
probably is not in the header ... did i write something about the header?

The idea was to get this number from the the front-end delivered but not sure whether that is possible
![image](https://user-images.githubusercontent.com/32402063/141318949-80f27783-675b-4249-b940-6c298325d938.png)

Reply:
I meant to write index.html. Might not be possible. We need to check on this